src/Juvix/Compiler/Nockma/Stdlib.hs

```diff
@@ -0,0 +1,301 @@
+{-# LANGUAGE QuasiQuotes #-}
+
+module Juvix.Compiler.Nockma.Stdlib where
+
+import Juvix.Compiler.Nockma.Translation.FromSource.QQ
+import Juvix.Prelude
+
+stdlib :: Term Natural
```

Comment:
This doesn't seem to be used anywhere. Is it used for compilation? I think it might be a pain to keep it up-to-date with the actual Nockma stdlib. Maybe a better solution is to have an extra instruction `StdLibCall` with an enum of functions and translate them to library calls only at serialization, and implement them directly in Haskell for the interpreter?